ntbs-service/Pages/Notifications/Edit/PatientDetails.cshtml.cs

```diff
@@ -115,17 +115,18 @@ protected override async Task<IActionResult> PrepareAndDisplayPageAsync(bool isB
                 return null;
             }
 
-            var ntbsNhsNumberMatches = await NotificationRepository.GetNotificationIdsByNhsNumberAsync(nhsNumber);
+            var ntbsNhsNumberMatches = NotificationRepository.GetNotificationIdsByNhsNumberAsync(nhsNumber);
+            var legacyNhsNumberMatches = _migrationRepository.GetLegacyNotificationNhsNumberMatches(nhsNumber);
+            await Task.WhenAll(legacyNhsNumberMatches, ntbsNhsNumberMatches);
```

Comment:
Rather than using `Task.WhenAll` and then `.Result` here (`.Result` is a bit of a code smell) you can also do:
```
var taskA = MethodA();
var taskB = MethodB();
resultA = await taskA;
resultB = await taskB;
```
which will have the same effect, as it will kick both tasks off at once and then wait for them to finish.

Reply:
Made this change 👍